tests/test_orders.py

```diff
@@ -1,22 +1,21 @@
 import requests
+import requests_mock
+
 from woocommerce import API
 
 from mock_data.mock_order import order
+from app import woo_get
+
+BASE_URL = "https://www.anything.com"
 
-def test_list_orders(mocker):
-    url = "https://www.anything.com"
-    ck = "test_key"
-    cs = "test_secret"
-    wcapi = API(
-        url=f"{url}",
-        consumer_key=f"{ck}",
-        consumer_secret=f"{cs}",
-        version="wc/v3"
-    )
-    mocker.patch('requests.get')
-    wcapi.get("/orders", params={"per_page": 20}).json()
+def test_list_orders():
+    with requests_mock.Mocker() as m:
```

Comment:
tried as decorator but pytest did not like passing `m` into the function ("fixture not found") so went with the context manager